.golangci.yml

```diff
@@ -0,0 +1,162 @@
+---
```

Comment:
I would remove all comments from this file.

Leaving just at the top of the file:
```yaml
# Ref: https://golangci-lint.run/usage/configuration/
# Inspired by https://github.com/ccoVeille/golangci-lint-config-examples/
```

For documentation and attribution.